cogs/web.py

```diff
@@ -4,16 +4,17 @@
 import aiohttp
 import discord
 import requests
+from discord.ext import commands, tasks
 from discord import app_commands
 from pymongo import MongoClient
 
 # database connection
 cluster = MongoClient(os.environ.get("mongo"))
 db = cluster["discord"]
-collection = db["status"]
+collection = db["status-web"]
 
 
-class Web(app_commands.Group):
+class Web(commands.GroupCog):
```

Comment:
I'm confused, what's the difference between app_commands.Group and commands.GroupCog?